src/jaxsim/api/contact.py

```diff
@@ -395,3 +406,96 @@ def mixed_jacobian(W_H_C: jtp.Matrix, W_J_WC: jtp.Matrix) -> jtp.Matrix:
             raise ValueError(output_vel_repr)
 
     return O_J_WC
+
+
+@jax_dataclasses.pytree_dataclass
+class ContactsState(JaxsimDataclass, abc.ABC):
+    """
+    Abstract class storing the state of the contacts model.
+    """
+
+    @classmethod
+    def build(cls, **kwargs) -> ContactsState:
+        """
+        Build the contact state object.
+        Returns:
+            The contact state object.
+        """
+
+        return cls(**kwargs)
+
+    @classmethod
+    def zero(cls, **kwargs) -> ContactsState:
+        """
+        Build a zero contact state.
+        Returns:
+            The zero contact state.
+        """
+
+        return cls.build(**kwargs)
+
+    def valid(self, **kwargs) -> bool:
+        """
+        Check if the contacts state is valid.
+        """
+
+        return True
+
+
+@jax_dataclasses.pytree_dataclass
+class ContactsParams(JaxsimDataclass, abc.ABC):
```

Comment:
Same comment as for `ContactState`.

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/178/commits/6542f8bb0cbd3d0607ba4db9a909740c8da63951